### PR TITLE
add product datapage and dashboard update

### PR DESCRIPTION
**Pull request by hanamantutagi**
updated datapage for product order.
removed top 10 IBO follow up from dashboard

### CODE REVIEW

File: addproduct.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h3 class="header-title">Add Product Order</h3>
             <div class="card">
                 <div class="card-body">
                     <div class="overflow-auto patient-registration">
-                        <script type="text/javascript" src="https://c1dce987.caspio.com/dp/2edda0008580a82fd7ce4ab68527/emb"></script>
+                        <script type="text/javascript" src="https://c1dce987.caspio.com/dp/2edda0005c71568357dc43e99f11"></script>
```
Comment (jjsg) [inline addproduct.html:28]: why no /emb at end?

File: productorderdetails.html [hunk 2]
```diff
@@ -25,7 +25,7 @@ <h3 class="header-title">Edit Product Order</h3>
             <div class="card">
                 <div class="card-body">
                     <div class="overflow-auto patient-registration">
-                        <script type="text/javascript" src="https://c1dce987.caspio.com/dp/2edda0001963c34401a942f58a47/emb"></script>
+                        <script type="text/javascript" src="https://c1dce987.caspio.com/dp/2edda000da33c7809bae43568cbf"></script>
```
Comment (jjsg) [inline productorderdetails.html:28]: same

File: ibo/ibodashboard.html [hunk 3]
```diff
@@ -18,14 +18,14 @@
         <div class="col-md-12">
             <h3 class="header-title">Dashboard</h3>
             <div class="card">
-                <div class="card-body">
+            <!--    <div class="card-body">
```
Comment (jjsg) [inline ibo/ibodashboard.html:21]: Removing the first chart, great

File: addproduct.html (hunk 1 above)
Comment (hanamantutagi) [reply addproduct.html:28]: corrected

**Pull request merged**